flexget/plugins/input/sickbeard.py

```diff
@@ -90,28 +100,27 @@ def on_task_input(self, task, config):
                 show_url = '%s:%s/api/%s/?cmd=show&tvdbid=%s' % (config['base_url'], config['port'],
                                                                  config['api_key'], show['tvdbid'])
                 show_json = task.requests.get(show_url).json()
-                sb_quality = show_json['data']['quality']
-                fg_quality = qualities[sb_quality]
+                fg_quality = self.quality_requirement_builder(show_json['data']['quality_details']['initial'])
             entry = Entry(title=show['show_name'],
                           url='',
                           series_name=show['show_name'],
                           tvdb_id=show.get('tvdbid'),
-                          tvrage_id=show.get('tvrage_id'),
-                          # configure_series plugin requires that all settings will have the configure_series prefix
-                          configure_series_quality=fg_quality)
+                          tvrage_id=show.get('tvrage_id'))
+            if len(fg_quality) > 1:
+                entry['configure_series_qualities'] = fg_quality
+            else:
+                entry['configure_series_quality'] = fg_quality[0]
```

Comment:
Fixed some stuff. The problem was when the quality parameter was not called at all and an empty string is being sent.